### PR TITLE
import custom certs in nexus truststore

### PR DESCRIPTION
**Pull request by muellerbe**
Closes https://github.com/osism/issues/issues/206

Signed-off-by: Bernd Mueller <mueller@b1-systems.de>

### CODE REVIEW

**Comment by berendt** [comment]
Please adjust the title of the PR and the commit message: https://docs.osism.tech/development/commit-messages.html#references-to-issues

File: roles/nexus/defaults/main.yml [hunk 1]
```diff
@@ -64,6 +64,8 @@ nexus_proxy_exclude_hosts:
   - "localhost"
   - "127.*"
   - "[::1]"
+nexus_security_ssl_url: "/service/rest/v1/security/ssl"
+nexus_security_truststore_url: "/service/rest/v1/security/ssl/truststore"
```
Comment (osfrickler) [inline roles/nexus/defaults/main.yml:68]: In which scenario would these need to be changed?

Or phrased differently: Do these really need to be role variables?

File: roles/nexus/tasks/import-certs.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+- name: Get PEM of URL
+  ansible.builtin.uri:
+    method: GET
+    url: "{{ nexus_api_url }}{{ nexus_security_ssl_url }}?host={{ item }}&port=443"
+    user: "{{ nexus_admin_username }}"
+    password: "{{ nexus_admin_password }}"
+    force_basic_auth: True
```
Comment (osfrickler) [inline roles/nexus/tasks/import-certs.yml:8]: Please address the yaml warnings.

**Review by osfrickler — CHANGES_REQUESTED** [review]

File: roles/nexus/defaults/main.yml (hunk 1 above)
Comment (muellerbe) [reply roles/nexus/defaults/main.yml:68]: do you mean, it could be hardcoded?

File: roles/nexus/tasks/import-certs.yml (hunk 2 above)
Comment (muellerbe) [reply roles/nexus/tasks/import-certs.yml]: done, thanks

File: roles/nexus/defaults/main.yml (hunk 1 above)
Comment (muellerbe) [reply roles/nexus/defaults/main.yml:68]: and I do not know, if there will be scenarios

**Pull request closed without merging**